src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsJWPlayerObject.java

```diff
@@ -14,13 +17,14 @@
   private static final Color COLOR_POSTROLL = new Color(253, 93, 167);
   private static final Color COLOR_VIDEO = new Color(0, 255, 13);
 
-  private static final String FEATURED_VIDEO_AD_SELECTOR = "#featured-video__player_ad";
+  private static final String FEATURED_VIDEO_AD_SELECTOR = "#featured-video__player iframe[src*=imasdk]";
```

Comment:
Is it more "prawilne" with such deep selector? ;)

Reply:
@fraszczakszymon points out it may not work on tablet and mobile in future

Reply:
I used "deeper" selector because for me (it could be false error) it wasn't recognized as a visible element